convert2rhel/unit_tests/checks_test.py

```diff
@@ -1432,6 +1432,100 @@ def test_is_loaded_kernel_latest_unsupported_skip(
 
         assert expected in caplog.records[-1].message
 
+
+@pytest.mark.parametrize(
+    (
+        "repoquery_version",
+        "return_code",
+        "package_name",
+        "unsupported_skip",
+        "allow_skip",
+        "expected",
+        "unsupported_log",
+    ),
+    (
+        (
+            "",
+            0,
+            "kernel-core",
+            "1",
+            "0",
+            "Detected 'CONVERT2RHEL_SKIP_KERNEL_CURRENCY_CHECK' environment variable, we will skip "
+            "the kernel-core comparison.\nBeware, this could leave your system in a broken state.",
+            "You are using the deprecated 'CONVERT2RHEL_UNSUPPORTED_SKIP_KERNEL_CURRENCY_CHECK'"
+            " environment variable.  Please switch to 'CONVERT2RHEL_SKIP_KERNEL_CURRENCY_CHECK'"
+            " instead.",
+        ),
+        (
+            "",
+            0,
+            "kernel-core",
+            "0",
+            "1",
+            "Detected 'CONVERT2RHEL_SKIP_KERNEL_CURRENCY_CHECK' environment variable, we will skip "
+            "the kernel-core comparison.\nBeware, this could leave your system in a broken state.",
+            None,
+        ),
+        (
+            "",
+            0,
+            "kernel-core",
+            "1",
+            "1",
+            "Detected 'CONVERT2RHEL_SKIP_KERNEL_CURRENCY_CHECK' environment variable, we will skip "
+            "the kernel-core comparison.\nBeware, this could leave your system in a broken state.",
+            "You are using the deprecated 'CONVERT2RHEL_UNSUPPORTED_SKIP_KERNEL_CURRENCY_CHECK'"
+            " environment variable.  Please switch to 'CONVERT2RHEL_SKIP_KERNEL_CURRENCY_CHECK'"
```

Comment:
Too many white spaces
```suggestion
            " environment variable. Please switch to 'CONVERT2RHEL_SKIP_KERNEL_CURRENCY_CHECK'"
```